deployment/localup/env.info

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # chain config
-CHAIN_ID="greenfield_9000-1741"
+CHAIN_ID="greenfield_9000-121"
```

Comment:
Why modity this?

Reply:
Because Greenfield repo has changed it into `greenfield_9000-121`

Reply:
The e2e test uses the specified greenfield tag,  why is there a change?